backend/internal/auth/bearer.go

```
@@ -84,29 +84,48 @@ func loadOrCreateEd25519Key(path string) (ed25519.PrivateKey, error) {
+		if err != nil {
+			return LoginResult{}, err
+		}
+	}
+	work.Release()
+
 	now := time.Now()
 	tx, err := s.db.BeginTx(ctx, nil)
 	if err != nil {
 		return LoginResult{}, err
 	}
 	defer tx.Rollback()
```

golangci-lint (2.13.2)

[error] Error return value of `tx.Rollback` is not checked

(errcheck)